gramps/gen/plug/docgen/treedoc.py

```diff
@@ -339,6 +344,12 @@ def write_start(self):
         self.write(0, '\\usepackage[%s,%s]{geometry}\n' % (paper, margin))
         self.write(0, '\\usepackage[all]{genealogytree}\n')
         self.write(0, '\\usepackage{color}\n')
+        trans = glocale.language[0][:2]
+        lang_en = _locale._get_language_string(trans).lower()
```

Comment:
The variable name `lang_en` implies English.  Can we call this `lang`?

Reply:
Ah yes, more clear.

Reply:
There is maybe a problem with code like:
IfFileExists{libertine.sty}! e.g., got an issue with 'shellesc', where " shellesc.sty"
does not exist anymore on recent TeX packages.
There is now (with recent TeX versions) something like compressed "doc and source"
into a "shellesc.dtx" file.
https://www.ctan.org/tex-archive/macros/latex/required/tools

'lmodern' should be a standard (ok, an old one)
https://www.ctan.org/pkg/lm

I did not try 'cm-super'.